src/app/features/auth/auth/login/auth-component.html

```diff
@@ -34,21 +41,31 @@ <h1 class="text-h2">Witaj</h1>
                 >
                 <mat-label>Password</mat-label>
```

Comment:
To trzeba by zmienić na 'Hasło', jako że reszta naszej aplikacji jest po polsku.

Reply:
zmieniłem